R/model_eligibility.R

```diff
@@ -561,14 +561,18 @@ calc_sd_check <- function(
   sd_check_plot_path = NULL,
   lookback_inside = 0,
   lookback_outside = 13,
+  lookback_inside_mean = 0,
+  lookback_outside_mean = 13,
+  lookback_inside_sd = 0,
+  lookback_outside_sd = 13,
   lookahead_inside = 1,
   lookahead_outside = 7,
   num_sd = 4,
   show_stats = FALSE
 ) {  
   ## get row index df and add end dates from target string 
   ## for subsetting and plotting
-  row_index <- attr(qfm, 'row_index') %>% 
+  row_index <- attr(qfm, 'row_index') <- attr(qfm, 'row_index') %>% 
```

Comment:
This seems unnecessary
```suggestion
  row_index <- attr(qfm, 'row_index') %>% 
```

Reply:
The reason I need it it is because the `as.data.frame` method looks at the `row_index` attribute for qfm and I need that to have `target_end_date` for the line plot.

Reply:
oh yeah :)